src/SpellCheck/SpellCheck.csproj

```diff
@@ -1,63 +1,52 @@
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <Description>Managed wrapper for the Microsoft Spell Checking API</Description>
+    <Copyright>Copyright © Bruce Bowyer-Smyth</Copyright>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Authors>Bruce Bowyer-Smyth</Authors>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/bbowyersmyth/SpellCheck</RepositoryUrl>
+    <PackageId>PlatformSpellCheck</PackageId>
+    <Product>Spell Check</Product>
+    <PackageProjectUrl>http://github.com/bbowyersmyth/spellcheck</PackageProjectUrl>
+    <PackageTags>spell checker check spell-check spell-checker spellchecker spellcheck suggestion</PackageTags>
+    <PackageReleaseNotes>UserDictionariesRegistrar, LanguageTag, Add, and AutoCorrect</PackageReleaseNotes>
+    <PackageLicenseExpression></PackageLicenseExpression>
+    <PackageLicenseFile>License.md</PackageLicenseFile>
     <RootNamespace>PlatformSpellCheck</RootNamespace>
     <AssemblyName>PlatformSpellCheck</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
-    <DocumentationFile>bin\Debug\PlatformSpellCheck.xml</DocumentationFile>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>C:\Users\WilliamForney\OneDrive - IMPROVGROUP LLC\Documents\GitHub\SpellCheck\src\SpellCheck\bin\$(Configuration)\PlatformSpellCheck.xml</DocumentationFile>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DocumentationFile>bin\Release\PlatformSpellCheck.xml</DocumentationFile>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>C:\Users\WilliamForney\OneDrive - IMPROVGROUP LLC\Documents\GitHub\SpellCheck\src\SpellCheck\bin\$(Configuration)\PlatformSpellCheck.xml</DocumentationFile>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.3" />
+  </ItemGroup>
+
   <ItemGroup>
     <Reference Include="MsSpellCheckLib">
       <HintPath>..\..\lib\MsSpellCheckLib.dll</HintPath>
-      <EmbedInteropTypes>True</EmbedInteropTypes>
```

Comment:
EmbedInteropTypes is important and the reason netstandard2.0 surface area can't be supported. I have another branch with a .net core conversion attempt that was waiting on netstandard2.1 or netcoreapp3.0 with the appropriate interop interfaces. Now that that has been released I'll complete that first.